src/main/java/uk/org/tombolo/importer/ons/CensusImporter.java

```diff
@@ -160,9 +160,15 @@ protected void importDatasource(Datasource datasource, List<String> geographySco
         return descriptions;
     }
 
-    private ArrayList<String> getDataSourceIDs() throws IOException {
-        return getSeedData().stream().map(CensusDescription::getDataSetTable)
-                .collect(Collectors.toCollection(ArrayList::new));
+    private List<String> getDataSourceIDs() {
+        try {
+            return getSeedData().stream().map(CensusDescription::getDataSetTable)
+                    .collect(Collectors.toCollection(ArrayList::new));
+        } catch (IOException e) {
+            log.error("An error has occurred while downloading DatasourceID's" + e.getMessage());
+        }
+
+        return null;
```

Comment:
I'd rather have an empty list here. If null is returned it will generate a NullPointerException in `CatalogueExportRunner` line 46.